src/edu/washington/escience/myria/expression/evaluate/PythonUDFEvaluator.java

```diff
@@ -97,22 +100,17 @@ public PythonUDFEvaluator(
    */
   private void initEvaluator() throws DbException {
     ExpressionOperator op = getExpression().getRootExpressionOperator();
-
     String pyFunctionName = ((PyUDFExpression) op).getName();
 
     try {
       if (pyFuncRegistrar != null) {
         FunctionStatus fs = pyFuncRegistrar.getFunctionStatus(pyFunctionName);
         if (fs != null && fs.getName() == null) {
```

Comment:
Wouldn't you just need `fs == null` here? If the function is not found, your `getFunctionStatus` method should return null, right?